.github/workflows/deploy-prod.yaml

```diff
@@ -74,5 +74,6 @@ jobs:
           -p MS_KEYCLOAK_CLIENT_ID=minespace-prod \
           -p SITEMINDER_URL=https://logon.gov.bc.ca \
           -p MATOMO_URL=https://matomo-4c2ba9-prod.apps.silver.devops.gov.bc.ca/ \
+          -p VCAUTHN_PRES_REQ_CONF_ID=minespace-access-0.1 \
```

Comment:
NOTE, this pres_req_conf_id doesn't exist yet. https://github.com/bcgov/a2a-trust-over-ip-configurations/tree/master/proof-configurations/minespace-access

If this version/solution needs to work in production, just copy paste the `test` one and change the id value to match this line.

Reply:
Oh, actually we would need to configure the VC-AUTHN installation to have a client for the PROD MDS Keycloak Realm. 

Let me know if you want me to do this before merging. But there are two steps before this can work in prod (permanent connection between vc-authn and mds Keycloak, and the proof configuration being added to the vc-authn deployment).

Reply:
I don't think we want to enable this on prod right now, so no need to set that up.